### PR TITLE
Replace Gin with http.ServeMux

### PR DESCRIPTION
**Pull request by djui**
An image speaks more than a thousands words.

Screenshot 2019-10-28 at 23 58 12

### CODE REVIEW

**Comment by Zatte** [comment]
Sure does! 
image

**Comment by Zatte** [comment]
Amazing stuff :)

**Comment by djui** [comment]
@Zatte See https://github.com/voi-go/svc/pull/6

File: options.go [hunk 1]
```diff
@@ -170,12 +149,13 @@ func WithPProfHandlers() Option {
 func WithHealthz() Option {
 	return func(s *SVC) error {
 		// Register live probe handler
-		s.Router.GET("/live", func(ctx *gin.Context) {
-			ctx.JSON(http.StatusOK, gin.H{"status": "Still Alive!"})
+		s.Router.HandleFunc("/live", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
+			_, _ = w.Write([]byte(`{"status": "Still Alive!"}`))
```
Comment (Zatte) [inline options.go]: Such magic constant

File: options.go [hunk 2]
```diff
@@ -186,10 +166,15 @@ func WithHealthz() Option {
 			}
 			if len(errs) > 0 {
 				s.logger.Warn("Ready check failed", zap.Any("errs", errs))
-				c.JSON(http.StatusServiceUnavailable, gin.H{"errors": errs})
-				return
+				b, err := json.Marshal(map[string]interface{}{"errors": errs})
```
Comment (K-Phoen) [inline options.go]: `s.logger.Warn("Ready check failed", zap.Any("errs", errs))` → `s.logger.Warn("Ready check failed", zap.Errors("errs", errs))`?

Comment (djui) [reply options.go]: Yes, but that will be my fourth PR ;-)

File: options.go (hunk 1 above)
Comment (djui) [reply options.go]: It's all about heap allocations.

**Pull request merged**